ui/dashboards/src/components/Dashboard/Dashboard.stories.tsx

```diff
@@ -532,16 +534,16 @@ const TIMESERIES_EXAMPLE_DASHBOARD_RESOURCE: DashboardResource = {
   },
 };
 
-const TIMESERIES_EXAMPLE_MOCK_NOW = 1673805600000;
-const TIMESERIES_EXAMPLE_MOCK_START = TIMESERIES_EXAMPLE_MOCK_NOW - 6 * 60 * 60 * 1000;
+const TIMESERIES_EXAMPLE_MOCK_END = 1673805600000;
```

Comment:
Decided to rename this while I was in here based on good feedback around similar mock times in the prior PR.